code/game/objects/structures/statues.dm

```diff
@@ -288,6 +288,31 @@
 		return
 	setDir(turn(dir, 90))
 
+/obj/structure/statue/kidanstatue
+	name = "Obsidian Kidan warrior statue"
+	desc = "A beautifully carved and menacing statue of a Kidan warrior made out of obsidian. It looks very heavy."
+	icon_state = "kidan"
+	anchored = TRUE
+	density = TRUE
```

Comment:
`density` is not required since it is already set on the parent. `anchored` is needed. Same for the ones below